articles/remapping.md

```diff
@@ -0,0 +1,77 @@
+### Remapping Names
+#### ROS 1
+In ROS 1 [Names](http://wiki.ros.org/Names) can be remapped to other names by passing [remapping arguments](http://wiki.ros.org/Remapping%20Arguments) via the command line. `roscpp` additionally allows remaps to be passed via Init as strings in STL containers. A ROS1 remap consists of two names: one that should be replaced with another.
+
+In ROS1 remapping works with FQN. When a name is to be checked for remappings, it is first expanded to a FQN. Both parts of the remapping rule are expanded to a FQN. Then a name is changed to the remap's second name only if it exactly matches the first name.
```

Comment:
I'll check roscpp's implementation and link it. I got the impression that's how it worked from the [third paragraph of this page](http://wiki.ros.org/Remapping%20Arguments). I think it makes a difference if: remapping `foo:=bar`, namespace: `ns`, names used in code `foo` `/ns/foo`. If done pre-FQN only the first name will get remapped; if done post-FQN both will get remapped. 

I'll update to make sure FQN is defined.